tests/resolvers/Chat/organization.spec.ts

```diff
@@ -0,0 +1,51 @@
+import "dotenv/config";
+import { organization as organizationResolver } from "../../../src/resolvers/Chat/organization";
+import { connect, disconnect } from "../../helpers/db";
+import type mongoose from "mongoose";
+import { Organization } from "../../../src/models";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import type { TestChatType } from "../../helpers/chat";
+import { createTestChatMessage } from "../../helpers/chat";
+
+let testChat: TestChatType;
+let MONGOOSE_INSTANCE: typeof mongoose;
+
+beforeAll(async () => {
+  MONGOOSE_INSTANCE = await connect();
+  const userOrgChat = await createTestChatMessage();
+  testChat = userOrgChat[2];
+});
+
+afterAll(async () => {
+  await disconnect(MONGOOSE_INSTANCE);
+});
+
+describe("resolvers -> Chat -> organization", () => {
+  it(`returns user object for parent.users`, async () => {
```

Comment:
The test description (`returns user object for parent.users`) is not accurate. You're testing the organization resolver, not the user resolver.  Update the test description to properly reflect the behavior you're testing:

```javascript
it("resolves the correct organization for the given chat", async () => {
  const parent = testChat?.toObject();
  if (!parent) {
    throw new Error("Parent object is undefined.");
  }

  const organizationPayload = await organizationResolver(parent, {}, {});

  const organization = await Organization.findOne({
    _id: testChat?.organization,
  }).lean();

  expect(organizationPayload).toEqual(organization);
});
```